### PR TITLE
VMDIRAC Settings for travis and basic unit test

### PR DESCRIPTION
**Pull request by myco**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+install:
+  - "pip install --upgrade setuptools"
+  - "pip install --upgrade pip"
+  - "pip install -r requirements.txt"
+# command to run tests
+script:
+  - cd ..
+  - wget https://github.com/DIRACGrid/DIRAC/archive/rel-v6r15.zip
+  - unzip -q rel-v6r15.zip
+  - mv DIRAC-rel-v6r15 DIRAC
```
Comment (fstagni) [inline .travis.yml]: I don't like hard-coding a version of DIRAC. If really it's need to do something like that, than you should use the content of "master" which corresponds to the "last production version". So I would probably use https://github.com/DIRACGrid/DIRAC/archive/master.zip

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+python-coveralls
+pytz>=2015.7
+readline>=6.2.4
+requests>=2.9.1
+rst2pdf>=0.93
+simplejson>=3.8.1
+six
+sqlalchemy>=1.0.9
+stomp.py>=3.1.5
+suds-jurko>=0.6
```
Comment (fstagni) [inline requirements.txt]: I am pretty sure you don't need most of these.

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+pytest-cov>=2.2.0
```
Comment (myco) [inline requirements.txt:1]: OK, I set a minimum requirements to pass tests.

**Comment by fstagni** [comment]
OK thanks. Just, please re-add the README.md file.

**Pull request merged**